string mplcommand
---
float64[] valid_path
